README.md

```diff
@@ -45,6 +45,31 @@ jobs:
 ```
 </details>
 
+### Jira Ticket Tagging
+
+<details>
+<summary>The action can be used to tag jira issues.</summary>
+
```

Comment:
We can add some more information hereabout the changelog process. Maybe a link to a Confluence page ....